app/views/shared/_comment.html.erb

```diff
@@ -0,0 +1,9 @@
+<% if (post.comments.empty?) %>
+  <p>No comment yet.</p>
+<% else %>
+  <% post.recent_comments.each do |comment| %>
+    <ul class="border-black ">
+      <li><%= comment.author.name %>: <%= comment.text %></li>
+    </ul>
+  <% end %>
+<% end %>
```

Comment:
- [ ] When I click on see all posts button, I got the following error, kindly fix it and make sure your app work properly
>![image](https://user-images.githubusercontent.com/53402600/222243211-ff4af184-b8bd-4e91-93a1-2f60e62023a3.png)

Reply:
HI @omarsalem7 , thank you for notice that, i will fix it